g3doc/Tutorials/Toy/Ch-1.md

```diff
@@ -22,8 +22,6 @@ This tutorial is divided in the following chapters:
     high level language specific semantics towards a generic linear-algebra
     oriented dialect for optimizations. Ultimately we will emit LLVM IR for code
     generation.
--   [Chapter #5](Ch-6.md): A REPL?
--   [Chapter #6](Ch-7.md): Custom backends? GPU using LLVM? TPU? XLA
```

Comment:
These one were intended.

Reply:
Oh, I see.

Should we rename chapter names? Now, chapter 5 is duplicated.

Reply:
Sure.

We're landing an update of the tutorial very soon though.